digital_benchmark/crawler_instagram/crawler_instagram/pipelines.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+# Define your item pipelines here
+#
+# Don't forget to add your pipeline to the ITEM_PIPELINES setting
+# See: https://docs.scrapy.org/en/latest/topics/item-pipeline.html
+import psycopg2
+import datetime
+time_now=datetime.datetime.utcnow()
+from scrapy.utils.project import get_project_settings
+settings = get_project_settings()
+
+
+class InstagramPipeline(object):
+
+    def open_spider(self, spider):
+        self.connection = psycopg2.connect(host=settings.get('HOSTNAME'), user=settings.get('USERNAME'), password=settings.get('PASSWORD'), dbname=settings.get('DB_NAME'))
+        self.cursor = self.connection.cursor()
+    
+    def close_spider(self, spider):
+        self._update_current_spyder()
+        self.cursor.close()
+        self.connection.close()
+
+
+    def process_item(self, item, spider):
+        self.unique_id_of_crawler_instance=item.get('unique_id')
+        self.insta_uid=item.get('insta_uid')
+        self.media_count=item.get('media_count')
+        insta_user_id=0
+        profile=self._user_exist(item.get('insta_uid'))
+        if profile:
+            insta_user_id=profile[0]
+        elif item.get('_type')=="profile":
+            self._parse_profile(item)
+        if item.get('_type')=="media":
+            self._parse_media(item,insta_user_id)
+        elif item.get('_type')=="comment":
+            self._parse_comment(item)
+        return item
+
+
+    def _user_exist(self,insta_uid):
```

Comment:
`user_exists` suggests that this returns a bool. Change the function name to _get_user_from_db